qa/tasks/cephfs/kernel_mount.py

```diff
@@ -155,7 +154,7 @@ def umount(self, force=False):
         self.mounted = False
         self.cleanup()
 
-    def umount_wait(self, force=False, require_clean=False, timeout=900):
+    def umount_wait(self, force=False, require_clean=False):
```

Comment:
This cause test to fail - https://pulpito.ceph.com/vshankar-2022-03-06_14:47:51-fs-wip-vshankar-testing-20220304-132102-testing-default-smithi/6724246/

```
2022-03-06T16:11:31.791 INFO:tasks.cephfs_test_runner:Test if client hangs on unmount after evicting the client. ... ERROR
2022-03-06T16:11:31.792 INFO:tasks.cephfs_test_runner:
2022-03-06T16:11:31.792 INFO:tasks.cephfs_test_runner:======================================================================
2022-03-06T16:11:31.792 INFO:tasks.cephfs_test_runner:ERROR: test_unmount_for_evicted_client (tasks.cephfs.test_client_recovery.TestClientRecovery)
2022-03-06T16:11:31.793 INFO:tasks.cephfs_test_runner:Test if client hangs on unmount after evicting the client.
2022-03-06T16:11:31.793 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2022-03-06T16:11:31.793 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2022-03-06T16:11:31.794 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_8385b895290cb6c9879a284d9aadc440d9b45e95/qa/tasks/cephfs/test_client_recovery.py", line 490, in test_unmount_for_evicted_client
2022-03-06T16:11:31.794 INFO:tasks.cephfs_test_runner:    self.mount_a.umount_wait(require_clean=True, timeout=30)
2022-03-06T16:11:31.794 INFO:tasks.cephfs_test_runner:TypeError: umount_wait() got an unexpected keyword argument 'timeout'
2022-03-06T16:11:31.795 INFO:tasks.cephfs_test_runner:
2022-03-06T16:11:31.795 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2022-03-06T16:11:31.795 INFO:tasks.cephfs_test_runner:Ran 20 tests in 1501.854s
```